libdnf/dnf-sack.h

```diff
@@ -96,6 +96,9 @@ gboolean     dnf_sack_set_arch              (DnfSack        *sack,
 void         dnf_sack_set_all_arch          (DnfSack        *sack,
                                              gboolean        all_arch);
 gboolean     dnf_sack_get_all_arch          (DnfSack        *sack);
+void         dnf_sack_set_allow_vendor_change(DnfSack       *sack,
```

Comment:
What about to store it in Goal Class. The question is also what about packagekit?